deploy-templates/values.yaml

```diff
@@ -452,6 +452,15 @@ edp-tekton:
 sso:
   # -- Install OAuth2-proxy and Keycloak CRs as a part of EDP deployment.
   enabled: false
+  # Enable if using the addons approach (see https://github.com/epam/edp-cluster-add-ons)
+  # for EDP installation and the extension-oidc is already installed.
+  # This prevents the creation of an additional Keycloak resource and secret.
```

Comment:
keycloakOperatorResources:
   # Create kind: Keycloak/ClusterKeycloak as a part of chart installation
  createKeycloakCR: true
  # Can be Keycloak or ClusterKeycloak. 
  kind: Keycloak
  # Name of kind: Keycloak/ClusterKeycloak CR;
  name: main